cmd/flags/flags.go

```diff
@@ -44,14 +44,26 @@ func AddConfigFileFlag(flagSet *flag.FlagSet) {
 }
 
 // TryLoadConfigFile initializes viper with config file specified as flag
-func TryLoadConfigFile(v *viper.Viper, logger *zap.Logger) {
+func TryLoadConfigFile(v *viper.Viper) error {
 	if file := v.GetString(configFile); file != "" {
 		v.SetConfigFile(file)
 		err := v.ReadInConfig()
 		if err != nil {
-			logger.Fatal("Error loading config file", zap.Error(err), zap.String(configFile, file))
+			errors.Wrap(err, fmt.Sprintf("Error loading config file %s", file))
```

Comment:
missing `return`?  And you can use `Wrapf` for extra formatting.

Reply:
:+1: ouch, the change from log Fatal :)